packages/l/ldc/pspec_x86_64.xml

```diff
@@ -731,12 +732,23 @@
             <Path fileType="header">/usr/include/dlang/ldc/std/windows/syserror.d</Path>
             <Path fileType="header">/usr/include/dlang/ldc/std/zip.d</Path>
             <Path fileType="header">/usr/include/dlang/ldc/std/zlib.d</Path>
+            <Path fileType="library">/usr/lib64/LLVMgold-ldc.so</Path>
             <Path fileType="library">/usr/lib64/ldc_rt.dso.o</Path>
             <Path fileType="library">/usr/lib64/libdruntime-ldc-debug-shared.so</Path>
             <Path fileType="library">/usr/lib64/libdruntime-ldc-debug.a</Path>
             <Path fileType="library">/usr/lib64/libdruntime-ldc-lto.a</Path>
             <Path fileType="library">/usr/lib64/libdruntime-ldc-shared.so</Path>
             <Path fileType="library">/usr/lib64/libdruntime-ldc.a</Path>
+            <Path fileType="library">/usr/lib64/libldc_rt.asan.a</Path>
```

Comment:
Are these static libs necessary at all?

Reply:
Yes, I believe they are used so that LDC can compile binaries that can use sanitizers. Previously, LDC didn't contain these files because it couldn't detect some directories due our arch name (it was trying to look for something `unknown-linux-gnu` while we have `solus-linux-gcc`).